client/src/features/admin/UpdateSessionEnvironmentButton.tsx

```diff
@@ -105,15 +105,11 @@ function UpdateSessionEnvironmentModal({
           environmentId: environment.id,
           container_image: data.container_image,
           name: data.name,
-          default_url: data.default_url.trim() ? data.default_url : "",
-          description: data.description.trim() ? data.description : "",
+          default_url: data.default_url?.trim() ? data.default_url : "",
```

Comment:
It seems this can be done here as well:
```suggestion
          default_url: data.default_url?.trim() || "",
```